moskito-backend/src/Serializer/CampsiteSerializer.php

```diff
@@ -0,0 +1,52 @@
+<?php 
+
+namespace App\Serializer;
+
+use App\Entity\Campsite;
+
+class CampsiteSerializer {
+    public function serialize($elements){
+        $Json = [];
+        if (is_array($elements)) {
+            foreach($elements as $element) {
+                $Json[] = [
+                    'name' => $element->getName(),
+                    'street' => $element->getStreet(),
+                    'postalCode' => $element->getPostalCode(),
+                    'place' => $element->getPlace(),
+                    'telephone' => $element->getTelephone(),
+                    'email' => $element->getEmail(),
+                    'coordinates' => $element->getCoordinates()
+                ];
+            }
+        } else {
+            $Json[] = [
+                'name' => $elements->getName(),
+                'street' => $elements->getStreet(),
+                'postalCode' => $elements->getPostalCode(),
+                'place' => $elements->getPlace(),
+                'telephone' => $elements->getTelephone(),
+                'email' => $elements->getEmail(),
+                'coordinates' => $elements->getCoordinates()
+            ];
+        }
+        
+        return \json_encode($Json);
+    }
+
+    public function deserialize($content){
+
+            $postData = \json_decode($content);
+            $classObject = new Campsite();
```

Comment:
$classObject => maybe $campsite as better name?

Reply:
Ich used a neutral naming because I will copy/paste the serializer for other entities and I don't want to rename everything.

Reply:
I decided to take a neutral name, as I have to copy/paste for other serializer